rpcs3/Emu/Io/usio.h

```diff
@@ -21,11 +21,11 @@ class usb_device_usio : public usb_device_emulated
 	void usio_read(u8 channel, u16 reg, u16 size);
 
 private:
-	bool test_on = false;
-	bool test_key_pressed = false;
-	bool coin_key_pressed = false;
+	bool test_on[2]{};
+	bool test_key_pressed[2]{};
+	bool coin_key_pressed[2]{};
+	le_t<u16> coin_counter[2]{};
 	bool is_used = false;
```

Comment:
I'd rather have a std array of two structs containing these as members

Reply:
(The four values that were duplicated)

Reply:
Done.